src/htdocs/css/_features.scss

```diff
@@ -46,9 +21,5 @@
 }
 
 .feature-image {
-  width: 75px;
-}
-
-.feature-item > a > .feature-title {
-  font-size: 1.1em;
+  width: 300px;
```

Comment:
I'm not sure you need this? There is already a `max-width: 100%;` on images. Do you just want the images to be the same size? Generally, this column will be below 300px anyway so it will shrink to be 100% of the column width.